spec/controllers/api/v3/appointments_controller_spec.rb

```diff
@@ -169,16 +169,21 @@ def create_record_list(n, options = {})
             before { request.env["HTTP_X_SYNC_REGION_ID"] = request_facility_group.id }
 
             context "when process_token's sync_region_id is empty" do
-              it "syncs facility group records from beginning of time" do
-                process_token = make_process_token(current_facility_processed_since: Time.current,
-                                                   other_facilities_processed_since: Time.current)
+              it "syncs facility group records" do
+                facility_group_records = [
```

Comment:
This is the culprit case: we shouldn't treat this as a special case since users _will_ receive `process_token.sync_region_id` in the next request and do a resync in following request if they need to.